hw/formal/formal_conn

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
```

Comment:
This batch script is temp support for user to run connectivity test locally with GUI. Once add the -GUI support in dvsim, we can remove this script.